diff --git b/vendor/k8s.io/kubernetes/cmd/kube-proxy/app/server.go a/vendor/k8s.io/kubernetes/cmd/kube-proxy/app/server.go
index 2f9ea2ea..35a7c14e 100644
--- b/vendor/k8s.io/kubernetes/cmd/kube-proxy/app/server.go
+++ a/vendor/k8s.io/kubernetes/cmd/kube-proxy/app/server.go
@@ -318,6 +318,10 @@ func (o *Options) ApplyDefaults(in *kubeproxyconfig.KubeProxyConfiguration) (*ku
 	return out, nil
 }
 
+func (o *Options) SetConfig(in *kubeproxyconfig.KubeProxyConfiguration) {
+	o.config = in
+}
+
 // NewProxyCommand creates a *cobra.Command object with default parameters
 func NewProxyCommand() *cobra.Command {
 	opts := NewOptions()
diff --git b/vendor/k8s.io/kubernetes/cmd/kube-proxy/app/server_others.go a/vendor/k8s.io/kubernetes/cmd/kube-proxy/app/server_others.go
index 380e83e2..51bb09fd 100644
--- b/vendor/k8s.io/kubernetes/cmd/kube-proxy/app/server_others.go
+++ a/vendor/k8s.io/kubernetes/cmd/kube-proxy/app/server_others.go
@@ -72,7 +72,7 @@ func newProxyServer(
 	if c, err := configz.New(proxyconfigapi.GroupName); err == nil {
 		c.Set(config)
 	} else {
-		return nil, fmt.Errorf("unable to register configz: %s", err)
+		glog.Warningf("unable to register configz: %s", err)
 	}
 
 	protocol := utiliptables.ProtocolIpv4
diff --git b/vendor/k8s.io/kubernetes/plugin/cmd/kube-scheduler/app/server.go a/vendor/k8s.io/kubernetes/plugin/cmd/kube-scheduler/app/server.go
index 93982f89..a116da41 100644
--- b/vendor/k8s.io/kubernetes/plugin/cmd/kube-scheduler/app/server.go
+++ a/vendor/k8s.io/kubernetes/plugin/cmd/kube-scheduler/app/server.go
@@ -380,7 +380,7 @@ func NewSchedulerServer(config *componentconfig.KubeSchedulerConfiguration, mast
 	if c, err := configz.New("componentconfig"); err == nil {
 		c.Set(config)
 	} else {
-		return nil, fmt.Errorf("unable to register configz: %s", err)
+		glog.Warningf("unable to register configz: %s", err)
 	}
 
 	// Prepare some Kube clients.
