diff --git a/vendor/k8s.io/kubernetes/cmd/kube-proxy/app/server_others.go b/vendor/k8s.io/kubernetes/cmd/kube-proxy/app/server_others.go
index 380e83e2a..51bb09fd0 100644
--- a/vendor/k8s.io/kubernetes/cmd/kube-proxy/app/server_others.go
+++ b/vendor/k8s.io/kubernetes/cmd/kube-proxy/app/server_others.go
@@ -72,7 +72,7 @@ func newProxyServer(
 	if c, err := configz.New(proxyconfigapi.GroupName); err == nil {
 		c.Set(config)
 	} else {
-		return nil, fmt.Errorf("unable to register configz: %s", err)
+		glog.Warningf("unable to register configz: %s", err)
 	}
 
 	protocol := utiliptables.ProtocolIpv4
