diff -ru gluster-4.1.5/rpc/xdr/gen/Makefile.am gluster-4.1.5.patched/rpc/xdr/gen/Makefile.am
--- gluster-4.1.5/rpc/xdr/gen/Makefile.am	2018-09-21 16:13:45.958611272 +0200
+++ gluster-4.1.5.patched/rpc/xdr/gen/Makefile.am	2018-09-24 19:50:32.798967652 +0200
@@ -19,14 +19,14 @@
 # in the build. Or we do this crufty thing instead.
 $(XDRSOURCES): $(XDRGENFILES)
 	@if [ ! -e $(xdrdst)/$@ -o $(@:.c=.x) -nt $(xdrdst)/$@ ]; then \
-		rpcgen -c -o $(xdrdst)/$@ $(@:.c=.x) ;\
+		/usr/bin/rpcgen -c -o $(xdrdst)/$@ $(@:.c=.x) ;\
 	fi
 
 # d*mn sed in netbsd6 doesn't do -i (inline)
 # (why are we still running smoke on netbsd6 and not netbsd7?)
 $(XDRHEADERS): $(XDRGENFILES)
 	@if [ ! -e $(xdrdst)/$@ -o $(@:.h=.x) -nt $(xdrdst)/$@ ]; then \
-		rpcgen -h -o $(@:.h=.tmp) $(@:.h=.x) && \
+		/usr/bin/rpcgen -h -o $(@:.h=.tmp) $(@:.h=.x) && \
 		sed -e '/#ifndef/ s/-/_/g' -e '/#define/ s/-/_/g' \
 			-e '/#endif/ s/-/_/' -e 's/TMP_/H_/g' \
 			$(@:.h=.tmp) > $(xdrdst)/$@ && \
Only in gluster-4.1.5.patched/rpc/xdr/gen: Makefile.am~
diff -ru gluster-4.1.5/rpc/xdr/gen/Makefile.in gluster-4.1.5.patched/rpc/xdr/gen/Makefile.in
--- gluster-4.1.5/rpc/xdr/gen/Makefile.in	2018-09-21 16:13:56.087638030 +0200
+++ gluster-4.1.5.patched/rpc/xdr/gen/Makefile.in	2018-09-24 19:51:06.198109046 +0200
@@ -558,14 +558,14 @@
 # in the build. Or we do this crufty thing instead.
 $(XDRSOURCES): $(XDRGENFILES)
 	@if [ ! -e $(xdrdst)/$@ -o $(@:.c=.x) -nt $(xdrdst)/$@ ]; then \
-		rpcgen -c -o $(xdrdst)/$@ $(@:.c=.x) ;\
+		/usr/bin/rpcgen -c -o $(xdrdst)/$@ $(@:.c=.x) ;\
 	fi
 
 # d*mn sed in netbsd6 doesn't do -i (inline)
 # (why are we still running smoke on netbsd6 and not netbsd7?)
 $(XDRHEADERS): $(XDRGENFILES)
 	@if [ ! -e $(xdrdst)/$@ -o $(@:.h=.x) -nt $(xdrdst)/$@ ]; then \
-		rpcgen -h -o $(@:.h=.tmp) $(@:.h=.x) && \
+		/usr/bin/rpcgen -h -o $(@:.h=.tmp) $(@:.h=.x) && \
 		sed -e '/#ifndef/ s/-/_/g' -e '/#define/ s/-/_/g' \
 			-e '/#endif/ s/-/_/' -e 's/TMP_/H_/g' \
 			$(@:.h=.tmp) > $(xdrdst)/$@ && \
Only in gluster-4.1.5.patched/rpc/xdr/gen: Makefile.in~
